addons/mail/static/src/components/partner_im_status_icon/partner_im_status_icon.xml

```diff
@@ -19,19 +19,19 @@
             >
                 <t t-if="props.hasBackground">
                     <i class="o_PartnerImStatusIcon_outerBackground fa fa-circle fa-stack-1x"/>
-                    <i class="o_PartnerImStatusIcon_innerBackground fa fa-circle fa-stack-1x"/>
+                    <i class="o_PartnerImStatusIcon_innerBackground fa fa-circle fa-stack-1x text-white"/>
                 </t>
                 <t t-if="partner.im_status === 'online'">
-                    <i class="o_PartnerImStatusIcon_icon o-online fa fa-circle fa-stack-1x" title="Online" role="img" aria-label="User is online"/>
+                    <i class="o_PartnerImStatusIcon_icon o-online fa fa-circle fa-stack-1x text-primary" title="Online" role="img" aria-label="User is online"/>
```

Comment:
This works fine on Enterprise but it returns a wrong color on Community (purple).
I don't think there's a class to handle `$o-enterprise-primary-color`. We could use something like `text-success` instead to have a main color for online status between Enterprise and Community, but not in this PR. It's something we should probably discuss with @stefanorigano 🙂